frontend/components/instructionsComponent/navigation/footer/index.tsx

```diff
@@ -1,23 +1,77 @@
-import styles from "./footer.module.css";
+import React from "react";
+import { Box, Flex, Text, Link, IconButton, Select } from "@chakra-ui/react";
+import { ExternalLinkIcon } from "@chakra-ui/icons";
+import { inter } from "@/app/fonts";
+import { FaSquareGithub } from "react-icons/fa6";
+import { getNetworkConfig } from "@/constants/networks";
 
-export default function Footer() {
+const Footer: React.FC = () => {
   return (
-    <div className={styles.footer}>
-      <div className={styles.icons_container}>
-        <div>
-          <a
-            href="https://github.com/alchemyplatform/create-web3-dapp"
-            target={"_blank"}
+    <Box as="footer" w="full">
+      <Flex
+        alignItems="center"
+        justifyContent="space-between"
+        p={8}
+        m={"0 20px"}
+        borderTop="1px"
+        borderColor="#000000"
+      >
+        {/* Left side: Lukso Mainnet and SC */}
+        <Flex alignItems="center">
+          <Box minWidth={"170"} mr="20px">
+            <Select
+              defaultValue={process.env.NEXT_PUBLIC_DEFAULT_NETWORK!}
+              onChange={(event) =>
+                (window.location.href = getNetworkConfig(
+                  event.target.value,
+                ).baseUrl)
+              }
+            >
+              <option value={"mainnet"}>LUKSO Mainnet</option>
+              <option value={"testnet"}>LUKSO Testnet</option>
+            </Select>
+          </Box>
+          <Text
+            fontSize="14px"
```

Comment:
should we use chakra sizing ("sm", "md" etc..) that uses em I think instead of pixels for more responsive design https://chakra-ui.com/docs/styled-system/responsive-styles

Reply:
@tehnalogos should we look at the sizes in the design or use 'sm' 'md', etc..?

Reply:
@Ledespana @democrazyfi yeah. think that would be good.